src/library_pthread.js

```diff
@@ -611,6 +611,11 @@ var LibraryPThread = {
     var schedPrio = 0;
     if (attr) {
       stackSize = {{{ makeGetValue('attr', 0, 'i32') }}};
+      // Musl has a convention that the stack size that is stored to the pthread attribute structure is always DEFAULT_STACK_SIZE smaller than the
+      // actual used stack size. That is, stored stack size of 0 would mean a stack of DEFAULT_STACK_SIZE in size. All musl functions hide this impl
+      // detail, and offset the size transparently, so pthread_*() API user does not operate in these offset sizes. When reading the structure directly,
+      // we need to offset the size manually here.
+      stackSize += 81920 /*DEFAULT_STACK_SIZE*/;
```

Comment:
to be sure i understand, this `DEFAULT_STACK_SIZE` thing is an internal musl detail? it's not a linux or an emscripten thing? if so maybe call it `musl's DEFAULT_STACK_SIZE` to avoid confusion?

Reply:
That's correct - good point, clarified the wording here.